src/components/Sheet/Sheet.tsx

```diff
@@ -70,13 +70,11 @@ export function Sheet({
     [mobile],
   );
 
-  useEffect(
-    () => {
-      handleResize();
-    },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [],
-  );
+  // eslint-disable react-hooks/exhaustive-deps
```

Comment:
What's the issue here? can you disable prettier instead? I'd feel better disabling the styling tool than the linter

Reply:
We were originally disabling the `react-hooks/exhaustive-deps` rule because we were explictly saying the dependency array is empty, while linting things it should contain `handleResize` and prettier didn't like having that disable-next-line comment in the middle of the code.

It looks like vscode not spotting linting config updates means the handleResize had snuck back in when I wasn't looking. After a vscode restart it seems like we can keep to ignoring the exhaustive deps rule without using the block formatting.

---

Not specifying this feels like a potential subtle bug but I'd like to keep this PR focused on formatting updates only